### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by HamidOna**
Added `run_pipeline` function for data preprocessing, model training, and evaluation.

Import necessary libraries before running the pipeline.

The `run_pipeline` function performs the following steps:
- Drops rows with missing values.
- Drops columns starting with 'Info_' except 'Info_cluster'.
- Selects specific features for analysis.
- Standardizes the dataset.
- Splits the lentivirus dataset into train and test sets.
- Concatenates the lentivirus train data with the training data.
- Trains Random Forest, Logistic Regression, and SVM models.
- Makes predictions on the test set.
- Calculates the Matthews Correlation Coefficient (MCC) for each model.

### CODE REVIEW

**Comment by fcampelo** [comment]
Hi, some comments on the code:

> def run_pipeline(df, lentivirus_data, class_weights):

Not critical, but for generality you may want to call the input parameter `organism_specific_data` or something similar, rather than `lentivirus_data` (we may want to test other pathogens later ;-) )

*****

>   columns_to_drop = df.filter(regex='^Info_(?!cluster)').columns
    df.drop(columns_to_drop, axis=1, inplace=True)
>
>    selected_features = ['Info_cluster','feat_esm1b_15','feat_esm1b_38','feat_esm1b_46','feat_esm1b_69','feat_esm1b_70',
         'feat_esm1b_76',
         'feat_esm1b_87',
         (...)
         'feat_esm1b_1274',
         'Class']
>
>   df = df[selected_features]
>
>    scaler = StandardScaler()
>    mask = df.columns != "Class"
>    df.loc[:, mask] = scaler.fit_transform(df.loc[:, mask])
>
>    lentivirus = lentivirus_data[selected_features]
>    mask = lentivirus.columns != "Class"
>    lentivirus.loc[:, mask] = scaler.transform(lentivirus.loc[:, mask])

All of the above can be done outside of the pipeline function: feature reduction, and general preprocessing etc. - everything, except perhaps class rebalancing of the training data, can be done separately, so that the inputs (both the `df` and the `organism_specific_data`) don't need to be repeatedly preprocessed (which would slow down the pipeline function). 

I think the pipeline function should only do:
- filter the general data (`df` - already fully preprocessed beforehand) using an input vector indicating which `info_organism_id`s should be included.
- Split the `organism_specific_data` (also already preprocessed beforehand)  into training and validation sets - you can just use `GroupShuffleSplit` with Info_cluster as the grouping variable. 
- merge of the filtered `df` with the **training** `organism_specific_data` 
- Class rebalancing (note: keep all observations from `organism_specific_data` and undersample the majority class of the filtered `df`, so that you reach class balance in the combined dataset
- Model fitting (one model - you can have multiple models available and select one based on an input parameter, or just have a single one available)
- Model assessment using the **validation** `organism_specific_data`
- Calculate and return the performance estimate of the model (e.g., MCC)

Cheers,
Felipe
